src/web/mjs/connectors/FuryoSquad.mjs

```diff
@@ -0,0 +1,50 @@
+import Connector from '../engine/Connector.mjs';
+
+export default class FuryoSquad extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'furyosquad';
+        super.label = 'Furyo Squad';
+        this.tags = [ 'manga', 'french', 'high-quality' ];
+        this.url = 'http://www.furyosquad.com/';
+    }
+
+    async _getMangas() {
+        const categories = ['/', '/en-cours', '/termines', '/stoppes'];
+
+        let request,data;
```

Comment:
May declare variables when they are initialized (inside the loop).
No need to declare variables outside for re-use, there is no noticeable performance/memory gain.